lib/rollout.rb

```diff
@@ -352,14 +352,26 @@ def initialize(storage, opts = {})
     @groups  = { all: lambda { |_user| true } }
   end
 
-  def activate(feature)
+  def activate(feature, uid = nil, comment = nil)
```

Comment:
I noticed some default params for the methods have spacing between the param and nil and others don't. It's probably best to stay consistent with one.

Reply:
Whoops. Thanks!